### ISSUE TITLE
Error: EXDEV: cross-device link not permitted

### ISSUE THREAD

**Issue by SebastianM-C** [julia-actions/add-julia-registry#1]
Hello,

I tried runnig the `add-julia-registry` in a self-hosted environment, but I got the following error
```
Run julia-actions/add-julia-registry@v1
/usr/bin/ssh-agent
SSH_AUTH_SOCK=/tmp/ssh-Rqq5e240pwJv/agent.3043019; export SSH_AUTH_SOCK;
SSH_AGENT_PID=3043021; export SSH_AGENT_PID;
echo Agent pid 3043021;
/usr/bin/ssh-add /tmp/tmp-3043000-xgn58KVZPhVx
Identity added: /tmp/tmp-3043000-xgn58KVZPhVx (private registry deploy key)
/usr/bin/ssh-keyscan github.com
# github.com:22 SSH-2.0-babeld-a3fae306
# github.com:22 SSH-2.0-babeld-a3fae306
# github.com:22 SSH-2.0-babeld-a3fae306
# github.com:22 SSH-2.0-babeld-a3fae306
# github.com:22 SSH-2.0-babeld-a3fae306
github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==
/usr/bin/git clone git@github.com:MyOrg/MyRegistry.git /tmp/tmp-3043000-W0JHbLbUSYpl
Cloning into '/tmp/tmp-3043000-W0JHbLbUSYpl'...
[Error: EXDEV: cross-device link not permitted, rename '/tmp/tmp-3043000-W0JHbLbUSYpl' -> '/home/username/.julia/registries/MyRegistry/tmp-3043000-W0JHbLbUSYpl'] {
  errno: -18,
  code: 'EXDEV',
  syscall: 'rename',
  path: '/tmp/tmp-3043000-W0JHbLbUSYpl',
  dest: '/home/username/.julia/registries/MyRegistry/tmp-3043000-W0JHbLbUSYpl'
}
```

**Comment by christopher-dG** [comment]
I guess it would help to know a bit about the machine you're running on. Are `/tmp` and `/home` on the same disk/partition?

**Comment by SebastianM-C** [comment]
From what I understand no. `/tmp` is on a `tmpfs` and `/home` is on a `/dev/sdbx` filesystem.

**Comment by christopher-dG** [comment]
Ok yeah makes sense. I'll figure out an alternative `mv`.

**Comment by SebastianM-C** [comment]
Fixed by #2
